# Controller Registration
ControllerInfo request

---
# response is true if value accepted, otherwise server ignore the request and response false
bool response